commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -1611,7 +1327,7 @@ public Complex cos() {
         // Define in terms of cosh
```

Comment:
This comment is not relevant any more. However it is missing from ComplexFunctions so you should move it there.